webserver/main.py

```diff
@@ -42,25 +47,49 @@ def do_POST(self):
         print("Received input: {}".format(body))
         body_as_string = body.decode("utf-8") 
 
-        # There are always 2 oponents in fencing
-        # Each will have their point stored in a separate file
-        # When each scores, an id will be sent to the scoring server
-        # which will then increment the counter for that oponent id
-        # The dashboard will be responsible for reading the scores
-        # and declaring a winner
+        # There are always 2 oponents in fencing. Each will have their point
+        # stored in a separate file. When each scores, an id will be sent to the
+        # scoring server which will then increment the counter for that oponent
+        # id The dashboard will be responsible for reading the scores and
+        # declaring a winner.
 
         # body comes in format id=<id>
         id = body_as_string.split("=")[1]
         fencer_score_file = 'fencer#' + id + "_score.txt"
-        print("Point scored from fencer#" + id)
+        print("Touch from fencer#" + id)
+
+        global lastScoredPointTime
+        global lastScoredPointId
+        current_time = datetime.datetime.now()
+        time_delta_in_milliseconds = (current_time - last_touch_time).total_seconds() * 1000
+
+        # Do not record a point if the touch came from same fencer within a
+        # second, or from another fencer within
+        # double_touch_milliseconds_threshold (review fencing rules)
+        ignore_point = False
+        double_touch_milliseconds_threshold = 40
```

Comment:
Can we move this to a module level variable... and maybe in ALL CAPS to indicate it's a constant?

Reply:
related - I wanted to make those a const, but not an easy way in python to do so. any suggestions?